Scripts/Singletons/GameStats.gd

```diff
@@ -2,6 +2,15 @@ extends Node
 
 signal robotDied
 
+#Keys: level paths (I.e. res://Scenes/Levels/Tutorial/blah)     
+#Values: true if level is completed, false if not
+var levelCompletion = {}
+
+#Check if we should play tutorial (I.e. if player exits level without completing, won't replay tutorial automatically)
+#Keys: level paths (like levelCompletion)     
+#Values: true if should play tutorial, false otherwise (I.e. if level completed or re-entered after not completing)
+var playTutorial = {}
+
```

Comment:
scene_array now holds scene keys so we should just update these comments to say that the keys are the same keys in the SceneSwapper dict

Reply:
didn't change this cause I feel like it's easier to have levelCompletion & playTutorial's keys be the full filepath rather than SceneSwapper.scene_path's keys